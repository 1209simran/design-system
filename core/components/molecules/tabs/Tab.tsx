import * as React from 'react';

export interface ITabProps {
  label: React.ReactElement;
}

const Tab: React.FunctionComponent<ITabProps> = props => {
  const {
    children,
  } = props;

  return (
    <React.Fragment>
      {children}
    </React.Fragment>
  );
};

export default Tab;
