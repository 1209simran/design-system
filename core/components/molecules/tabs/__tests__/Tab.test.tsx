import * as React from 'react';
import { shallow } from 'enzyme';
import Tab from '../Tab';

describe('Tab component', () => {
  it('with Label', () => {
    const tree = shallow(
      <Tab label={<></>}>Tab</Tab>
    );
    expect(tree).toMatchSnapshot();
  });
});
