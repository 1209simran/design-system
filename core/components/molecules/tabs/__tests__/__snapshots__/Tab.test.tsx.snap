// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab component with Label 1`] = `
<Fragment>
  Tab
</Fragment>
`;
